src/Common/src/Interop/Unix/System.Native/Interop.readdir.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Sys
+    {
+        internal enum NodeType : short
+        {
+            Unknown         = 0,
```

Comment:
For constants, I've been keeping the names as defined by the platform headers on the C# side and prepending PAL_ on the C++ fide to avoid conflicts.